.devops/pagopa-code-review-pipelines.yml

```
@@ -76,7 +76,7 @@ resources:
     - repository: checkout-be-mock
       type: github
       name: pagopa/pagopa-checkout-be-mock
-      ref: master
+      ref: CHK-4524-mock-wallet-end-point
```

TODO: to be restored
```suggestion
      ref: master
```